docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+services:
+  dev:
+    build:
+      context: .
+      target: production
+    ports:
+      - "5000:5000"
+    volumes:
+      - ./todo_app:/todo-app/todo_app
+    env_file:
+      - ./.env
+    
+
+  prod:
+    build:
+      context: .
+      target: development
```

Comment:
The service names dev/prod are the wrong way around compared to what they're actually targeting (production/development respectively)